DEPS

```diff
@@ -467,6 +467,17 @@ deps = {
      'dep_type': 'cipd',
    },
 
+  'src/third_party/android_tools/sdk/licenses': {
+     'packages': [
+       {
+        'package': 'flutter/android/sdk/licenses',
+        'version': 'last_updated:2020-11-16T13:56:24-0800'
```

Comment:
Should we tag it with a ref or tag instead of using the date? E.g something more relevant to the state of the package?

Reply:
if we believe we need only one package in prod at a time then we add a ref release to the cipd package and use it here.